book/src/01_intro.md

```diff
@@ -60,31 +60,46 @@ public class Npc extends Character {
 Now you have stuff duplicated; you would have to write mostly identical code for
 your player and the NPC, even though e.g. they both share a transform.
 
-This is where ECS comes into play: Components are *associated* with entities;
-you just insert some component, whenever you like. Also, you only define the functionality
-once: One System, taking a `Force` and a `Mass` - it produces a `Velocity`.
+<img src="./images/entity-component.svg" alt="Entity-component relationship" width="20%" style="float:left;margin-right:15px" />
 
-In fact, an entity does not even own the components; it's just
+This is where ECS comes into play: Components are *associated* with entities; you just insert some component, whenever you like.
+One entity may or may not have a certain component. You can see an `Entity` as an ID into component tables, as illustrated in the
```

Comment:
s/component/component associated with it/

Reply:
That's what I had initially, but then the sentence sounded a bit unnatural and I didn't want to repeat "associated" as it's already used in the line before.

Reply:
I'm with @torkleyy here, the extra words don't add to comprehension.